### PR TITLE
devices: ibm-dpi os discovery

### PR DESCRIPTION
**Pull request by vivia11**
Hello all,

Tested with model IBM switched C19/C13 .

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/polling/os/ibm-dpi.inc.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$version = end(explode(' ', trim(snmp_get($device, "sysDescr.0", "-OQv", "SNMPv2-MIB"))));
```
Comment (laf) [inline includes/polling/os/ibm-dpi.inc.php]: We already have sysDescr value in `$device['sysDescr']`

Comment (vivia11) [reply includes/polling/os/ibm-dpi.inc.php]: Mine are rebranded Lenovo PDUs actually. Maybe they're all the same now?

No MIB files that I can find. I've emailed IBM/Lenovo and we'll see if there are even any.

**Comment by vivia11** [comment]
Mine are rebranded Lenovo PDUs actually. Maybe they're all the same now?

No MIB files that I can find. I've emailed IBM/Lenovo and hopefully they have MIBs for this (which I think they do).

**Comment by laf** [comment]
This might be better to call it just dpi.

**Comment by vivia11** [comment]
@laf 
After some digging, this is a model that is still owned by IBM, they apparently sold the other models of this (to Lenovo) type but withdrew this one and didn't sell it.

There are also no MIBs for this, confirmed by IBM.

**Pull request merged**